android/src/main/java/com/adyenreactnativesdk/component/dropin/DropInModule.kt

```diff
@@ -139,6 +158,7 @@ class DropInModule(context: ReactApplicationContext?) : BaseModule(context),
     }
 
     override fun onCompleted(result: String) {
+        // TODO: check voucher's use-case
```

Comment:
Do you still want to test vouchers?

Reply:
Done [6ee30cc](https://github.com/Adyen/adyen-react-native/pull/331/commits/6ee30cc4f03efe28e2d5debf707d6cac5b453c46)